KebPOS/Validation.cs

```diff
@@ -21,4 +25,37 @@ public static bool IsValidOrderId(int id, IEnumerable<Models.Order> orders)
         else
             return true;
     }
+
+    public static bool CheckStringLength(string stringToCheck, int nameLengthLimit)
+    {
+        bool valid = false;
```

Comment:
I guess this method is a bit redundant since this check is a one-liner. My bad. 🤦🏽‍♂️

![image](https://github.com/TheCSharpAcademy/CONSOLE.PointOfSale/assets/108711476/b5a2b572-5740-4b8c-b34a-a0ee11736bf3)